Plots/Contours/NCL_h_lat_6.py

```diff
@@ -0,0 +1,120 @@
+"""
+NCL_conOncon_1.py
```

Comment:
```suggestion
NCL_h_lat_6.py
```

Reply:
Thanks for catching these. I always start by recycling some code from a similar example and lately I've been forgetting to change the names.